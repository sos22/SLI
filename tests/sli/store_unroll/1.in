depth = 5
A true
Roots:
A
End of roots
