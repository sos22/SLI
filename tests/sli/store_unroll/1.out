A true
Roots:
A
End of roots
